### PR TITLE
[LRN] Add extra math symbols

### PR DESCRIPTION
**Pull request by karoltarasiuk**
\nparallel
\measuredangle
\overrightarrow
\overleftarrow

Conflicts:
	src/css/math.less

### CODE REVIEW

File: src/css/math.less [hunk 1]
```diff
@@ -298,9 +298,27 @@
...
   }
 
+  .mq-overarrow {
+    border-top: 1px solid black;
+    margin-top: 1px;
+    padding-top: 0.2em;
 
-  &, .mq-editable-field {
-    cursor: text;
-    font-family: @symbola;
```
Comment (laughinghan) [inline src/css/math.less]: This appears to remove Symbola from all of `.mq-math-mode` (that's what the `&` refers to in this context)

**Comment by laughinghan** [comment]
Thanks, this looks useful! Can you fix what appears to be a merge conflict?

**Comment by karoltarasiuk** [comment]
@laughinghan yeah - I totally missed it, for some reason didn't look at the diff carefully - it's ok now

File: src/css/math.less [hunk 2]
```diff
@@ -303,4 +303,28 @@
...
+      content: '\27A4';
+      text-align: right;
+    }
+    &.mq-arrow-left:before {
+      -moz-transform: scaleX(-1);
+      -o-transform: scaleX(-1);
+      -webkit-transform: scaleX(-1);
+      transform: scaleX(-1);
+      filter: FlipH;
+      -ms-filter: "FlipH";
```
Comment (laughinghan) [inline src/css/math.less]: LOL

**Comment by stufflebear** [comment]
+1

File: src/css/math.less (hunk 2 above)
Comment (karoltarasiuk) [inline src/css/math.less]: Believe me or not, we couldn't find the opposite arrow which looked exactly the same...

This is the right one: http://www.fileformat.info/info/unicode/char/27a4/index.htm It's called BLACK RIGHTWARDS ARROWHEAD. LEFTWARDS one doesn't exist...

Comment (laughinghan) [inline src/css/math.less]: I heard they considered including it, but were afraid there wasn't enough
space for the taco emoji

Han

Comment (karoltarasiuk) [inline src/css/math.less]: LOL that would be it I bet!

**Pull request merged**